File: app/src/main/java/com/if26/leuks/safelock/SplashScreenActivity.java
package com.if26.leuks.safelock;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

/**
 * Created by leuks on 16/11/2017.
 */

public class SplashScreenActivity extends AppCompatActivity{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent intent = new Intent(this, LoginActivity.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/if26/leuks/safelock/SplashScreenActivity.kt
package com.if26.leuks.safelock

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.if26.leuks.safelock.db.DbManager
import android.R.id.edit
import android.content.Context
import android.content.SharedPreferences
import android.content.Context.MODE_PRIVATE
import com.if26.leuks.safelock.tool.Const


/**
 * Created by leuks on 16/11/2017.
 */
class SplashScreenActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        //Init database
        DbManager.getInstance(this)

        ActivityManager.pin(this)
    }
}